libs/MotionKit/source/StabilizationControl.cpp

```diff
@@ -0,0 +1,58 @@
+// Leka - LekaOS
+// Copyright 2023 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "StabilizationControl.hpp"
+#include <algorithm>
+
+#include "MathUtils.h"
+
+using namespace leka;
+
+void StabilizationControl::init(EulerAngles starting_angle)
+{
+	_euler_angles_target = starting_angle;
+}
+
+auto StabilizationControl::processStabilizationAngle(EulerAngles current_angles) -> std::tuple<float, Rotation>
```

Comment:
I wonder if this control really needs a PID to achieve its purpose, but it's ok as is.